app/components/school_information/your_schools/participants_school_search_component.html.erb

```diff
@@ -0,0 +1,38 @@
+<div class="govuk-grid-row">
+  <%= form_with model: @school_search_form, url: lead_providers_your_schools_path, builder: GOVUKDesignSystemFormBuilder::FormBuilder, method: :post do |f| %>
+    <div class="govuk-grid-column-one-quarter index-group">
+      <div class="govuk-form-group govuk-!-margin-bottom-0">
+        <%= f.govuk_text_field :school_name, class: 'govuk-input govuk-!-margin-bottom-3', label: { text: "Search your schools", class: 'govuk-label govuk-label--s' } %>
+        <%= f.hidden_field :cohort_year, value: @selected_cohort.start_year %>
+        <%= f.hidden_field :selected_cohort_id, value: @selected_cohort.id %>
+        <%= f.govuk_submit "Search", classes: 'govuk-button govuk-!-margin-bottom-0' %>
+      </div>
+    </div>
+  <% end %>
+
+  <div class="govuk-grid-column-three-quarters">
+    <table class="govuk-table">
+      <thead class="govuk-table__head">
+        <tr class="govuk-table__row">
+          <th scope="col" class="govuk-table__header">Name</th>
+          <th scope="col" class="govuk-table__header">MAT / Network</th>
+          <th scope="col" class="govuk-table__header govuk-table__header--numeric"><%= @selected_cohort.start_year %>
+            participants
+          </th>
+        </tr>
+      </thead>
+      <tbody class="govuk-table__body">
+        <% @schools.each do |school| %>
```

Comment:
Worth showing a message when @schools is empty

Reply:
I agree – have a chat with Al about it

Reply:
Good point

Reply:
Done